code-challenges.test.js

```diff
@@ -25,27 +25,59 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe("divisibleByThree", () => {
+    const object1 = { number: 15 }
+    // Expected output: "15 is divisible by three"
+    const object2 = { number: 0 }
+    // Expected output: "0 is divisible by three"
+    const object3 = { number: -7 }
+    // Expected output: "-7 is not divisible by three"
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", ()=> {
+        expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+        expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+        expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+    })
+})
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+//input: object
+//output: a string that says the number is divisible by three
+const divisibleByThree = (object) => {
+    // set if/else conditions to check truthy or falsely
+    if(object.number % 3 === 0){
+        // return the string if the if condition is true
+        return `${object.number} is divisible by three`
```

Comment:
Would be good to mention that the condition is checking if the number is evenly divisible by 3